app/components/developers/badge_component.rb

```diff
@@ -0,0 +1,14 @@
+module Developers
+  class BadgeComponent < ApplicationComponent
+    private attr_reader :title, :classes
+
+    def initialize(title, classes)
+      @title = title
+      @classes = classes
+    end
+
+    def call
+      tag.span title, class: classes.to_s
```

Comment:
Jumping in before you marked this as ready for review, so apologies if you planned on making this change already.

I think this component could be more generic. `BadgeComponent`, for example, and you only pass in `title` and and an identifier to set the color classes. That would move the boilerplate CSS classes to this file and not require us to duplicate them.

We could also reconcile it with `Admin::BadgeComponent` since they would be the same thing!

Reply:
Sure. That makes more sense. I will do the necessary changes. Thanks for the feedback

Reply:
Just one point here I had initially coded to pass only bg-color and text-color as parameters and had added other classes added here directly but on second thought I felt that may be sometime you want to even customize other styles like font so I ended up passing all styles. Will change now to pass only color and we can again change in future if we need to customize any other styles.

Reply:
Good thinking! I like to optimize for what we need now. We can always refactor to override fonts in the future.